src/app/container/register-tool/register-tool.component.ts

```diff
@@ -72,7 +72,7 @@ export class RegisterToolComponent implements OnInit, AfterViewChecked, OnDestro
     {
       label: 'Register using GitHub Apps (Recommended)',
       extendedLabel:
-        'Install our GitHub App on your repository/organization to automatically sync tools with GitHub. Allows you to register a tool descriptor without linking to a Docker image you own.',
+        'Install our GitHub App on your repository in your personal account or organization to automatically sync tools with GitHub. Allows you to register a tool descriptor without linking to a Docker image you own.',
```

Comment:
Seems like there are two options:
1) On an personal organization 
2) On a regular organization
3) On a personal repo 
4) On a regular repo

Before I think it listed all but assumed the user knew that the personal org/repo was just like any other.
 
The new one is a little ambiguous, is it "your repository in your personal account or (personal) organization" (which skips 4 and 2) or is it "your repository in your personal account or (non-personal) organization" (which skips 1 and 3)

May be simpler as 
"Install our GitHub App on your repository/organization (whether personal or not)" or something better

Reply:
I may be missing some background or mixing up terminology, but what's the difference between a personal organization vs a regular one and a personal repo vs a regular repo? 

I worded it like so in the PR because after selecting where to install the app, the configuration page on GitHub is more focused on configuring the repository access

Reply:
I don't know if these are official terminology, but I'm interpreting `in your personal account or organization ` in the PR with:
My personal org is https://github.com/denis-yuen
A personal repo is https://github.com/denis-yuen/galaxy-workflow-dockstore-example-2
A regular org is https://github.com/dockstore
A regular repo is https://github.com/dockstore/compose_setup
Otherwise, I don't think there are any real differences  aside from a few subtleties when it comes to permissions , which is my I'm guessing the original text didn't emphasize "personal" at all.

Reply:
https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts

So in general I would suggest "personal account" and "organization", even though they say "organization account". In this case, how about dodging the issue a little bit and say "Install our GitHub App in your personal account or organization", and let the finer details of whether to install on individual repos or the whole account/org be left to the GitHub UI?

Reply:
Second Charles' idea.  In github's terminology, I believe that https://github.com/denis-yuen is a "personal account", and https://github.com/dockstore is an "organization".

Reply:
> In github's terminology, I believe that https://github.com/denis-yuen is a "personal account", and https://github.com/dockstore is an "organization account"

That's how I understood it as well. GitHub documents how to install the GH App on a [personal account](https://docs.github.com/en/apps/using-github-apps/installing-a-github-app-in-your-personal-account) or an [organization](https://docs.github.com/en/apps/using-github-apps/installing-a-github-app-in-your-organization) so I'll implement Charles' idea